### DISCUSSION TITLE
What Operating System do you run IPFilter on?

### DISCUSSION THREAD

**Discussion by DavidMoore** [DavidMoore/ipfilter#87] [Polls]
Hi

So I can make some decisions on new version and rewrite, what Operating System(s) do you use?

**Comment by ghost** [comment]
I think that you should support Windows from at least 7 onwards, but it would be nice if was compatible with windows XP SP3+ too. I wouldn't spend my time on Mac, but Linux is also appreciated (AppImage maybe)
